uv/reporter/base.py

```diff
@@ -18,7 +18,7 @@
 
```

Comment:
Changes here are for mypy to accomodate our new support for nested dictionaries as params.